### PR TITLE
Project Labyrinth - Andrea Larsson & Julia Ros

### PR DESCRIPTION
**Pull request by Andrealarsson**
Link to my deployed project:
https://labyrinth-game-app.netlify.app/ 

Things I'd like to have clarified or explained in more detail:
We would like some explanation of why our Go Back button doesn't work as we want? The user can go one step back, but the main description doesn't change, like the rest of the text when moving back. What is missing?

### CODE REVIEW

File: code/src/reducers/games.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        body: JSON.stringify({
+          username: getState().games.username
+        })
+      })
+      .then(res => {
+        if (res.ok) {
+          dispatch(games.actions.setError(null))
+          return res.json()
+        } else {
+          console.log(res)
```
Comment (CarolineF3) [inline code/src/reducers/games.js:67]: Remove this :)

Comment (CarolineF3) [inline code/src/reducers/games.js:67]: Remove this :)

File: code/src/reducers/games.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+          type: 'move',
+          direction: getState().games.direction
+        })
+      })
+      .then(res => {
+        if (res.ok) {
+          dispatch(games.actions.setError(null))
+          return res.json()
+        } else {
+          console.log(res)
```
Comment (CarolineF3) [inline code/src/reducers/games.js:100]: Remove this :)

File: code/src/components/GameScreen.js [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+            <Diamond />
+          )}
+        </OuterContainer>
+      </Background>
+  )
+}
+
+export default GameScreen
+
+const Background = styled.div`
```
Comment (CarolineF3) [inline code/src/components/GameScreen.js:67]: Good job using styled components! :)

File: code/src/components/GameScreen.js [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+            <Button 
+              value={move.direction}
+              onClick={(event) => onChooseDirection(event)}>
+                Go {move.direction}
+            </Button>
+          </InnerContainer>
+          ))
+          }  
+          {coordinates === '1,3' && (
+            <Diamond />
```
Comment (CarolineF3) [inline code/src/components/GameScreen.js:58]: Not sure if this is wrong, but it looks like Diamond is a component.  Or is it just av div with a missing closing tag?

File: code/src/index.css [hunk 5]
```diff
@@ -8,6 +8,5 @@ body {
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+  font-family: 'Barlow Semi Condensed', sans-serif;
```
Comment (CarolineF3) [inline code/src/index.css:11]: You could have added font-color here as well (instead of putting in in the components) in order to shorten the code. :)

**Review by CarolineF3 — APPROVED** [review]
Overview:
The game is working as it should 🙌, except for the Go Back button that is not entirely done 😃. Nice and clean design with beautiful background images, good take with a different image for mobile view. We like the Restart button that is available on all screens. 🎉🎉

**Review by KarolCislowski — APPROVED** [review]
Nice app! 
Please remember to ALWAYS remove all console logs before submitting the project.

**Pull request closed without merging**